### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by aguillenv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+Update:
+```bash
+brew update
+brew upgrade <formula>
+```
+
+## Available formulae
+- [anycloud](https://github.com/alantech/anycloud)
```
Comment (dfellis) [inline README.md]: Should we add `alan` to this, as well?

Comment (aguillenv) [reply README.md]: I've added it with checkboxes since it is not implemented yet, what do you think?

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 Alan Technologies, Inc
```
Comment (dfellis) [inline LICENSE]: 2021

Comment (aguillenv) [reply LICENSE]: Good one, I've copied from anycloud 😄

Comment (dfellis) [reply LICENSE]: Oh, then it's wrong there, too. We didn't start these until this year.

**Review by dfellis — APPROVED** [review]

**Review by depombo — APPROVED** [review]

**Pull request merged**